org.telegram.desktop.json

```diff
@@ -98,91 +87,173 @@
                 }
             ]
         },
+        {
+            "name": "cmake",
```

Comment:
Why do you build your own CMake? It's definitely in the runtime.

Reply:
Telegram wants cmake 3.16, while in the runtime there is 3.15.x

Reply:
Is there a particular reason why this version is needed? If they don't use any specific feature we can patch it and use cmake from the runtime.

Reply:
I don't know, probably? Anyway, I believe building own cmake module is way easier than maintaining yet another patch.